documentation/DS-documentation/dashboard.md

```diff
@@ -0,0 +1,98 @@
+You hope to see that the two lines track on top of each other because that means no messages are erroring. 
+If there are often gaps between the lines then it means a fraction of your jobs are erroring and you’ll need to figure out why (see MemoryUtilization and Show Errors or look directly in your Cloudwatch Logs for insights).
+
+## MemoryUtilization
+
+![Memory Utilization](images/memoryutilization.png)
+
+Insufficient memory is the error that we most often encounter (as we try to use the smallest machines possible for economy’s sake) so we like to look at memory usage. 
+Note that this is showing memory utilization in bulk for your cluster, not for individual machines. 
+Because different machines reach memory intensive steps at different points in time, and because we’re looking at an average across 5 minute windows, the max percentage you see is likely to be much less than 100%, even if you are using all the memory in your machines at some points.
+
+# MessagesVisible/NotVisible
+
+![MessagesVisible/NotVisible](images/messages_change_slope.png)
+
+Visible messages are messages waiting in your queue. 
+Hidden messages (aka MessagesNotVisible) have been started and will remain hidden until either they are completed and therefore removed from the queue or they reach the time set in SQS_MESSAGE_VISIBILITY in your config file, whichever comes first. 
+([Read more about Message Visibility](SQS_QUEUE_information.md).) 
+After starting your fleet (and waiting long enough for at least one round of jobs to complete), you hope to see a linear decline in total messages with the number of hidden messages equal to the number of jobs being run (fleet size * tasks per machine * docker cores).
+
+![Blip in MessagesVisible/NotVisible](images/blip_in_messagesnotvisible.png)
+
+Sometimes you’ll see a blip where there is a rapid increase in the number of hidden messages (as pictured above). 
+This can happen if there is an error on a machine and the hard disk gets full - it rapidly pulls jobs and puts them back until the machine error is caught and rebooted. 
+This type of error shows in this widget as it happens.
+
+If your spot fleet loses capacity (see FulfilledCapacity), you may see a blip in MessagesVisible/NotVisible where the number of hidden messages rapidly decreases. 
+This appears in the widget the amount of time set in SQS_MESSAGE_VISIBILITY in your config file after the capacity loss when jobs that were started (i.e. hidden) but not completed return to visible status.
+
+The relative slope of your graph can also be informative. 
+For the run pictured at top, we discovered that a fraction of our jobs were erroring because the machines were running out of memory. 
+Midway through 7/12 we upped the memory of the machines in our fleet and you can see from that point on a greater slope as more jobs were finishing in the same amount of time (as fewer were failing to complete because of memory errors.)
+
+## Distinct Logs with “cellprofiler -c”
```

Comment:
We don't actually want cellprofiler -c in the template - can we just have this be something like query_cmd or something like that, here and in the run.py file and throughout? We'll also need to add it as a thing to change in making your app.

Reply:
good catch. fixed here and in run.py and added to documentation that it must be customized